homeassistant/components/life360/button.py

```diff
@@ -0,0 +1,56 @@
+"""Support for Life360 buttons."""
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import Life360DataUpdateCoordinator
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Life360 buttons."""
+    coordinator: Life360DataUpdateCoordinator = hass.data[DOMAIN].coordinators[
+        config_entry.entry_id
+    ]
+    for member_id, member in coordinator.data.members.items():
+        async_add_entities(
```

Comment:
Please collect the entities in a list first and then make a single call to `async_add_entities`.